### DISCUSSION TITLE
请问如何调整页面宽度和引入外部字体文件呢？

### DISCUSSION THREAD

**Discussion by Xm798** [hugo-fixit/FixIt#80] [Q&A] [answered]
十分感谢您的工作！但由于对前端完全不懂，想请教两个小白问题：

1. 想要比较窄一点的页面宽度，要如何调整呢？想要类似于原 Loveit 的稍窄的页面宽度，包括文章页面和主页。
2. 应当如何引入外部字体文件？在 _override.scss 中找到了字体变量，但外部字体文件应当在何处引入呢？

谢谢！

**Comment by Lruihao** [comment, marked as answer]
自定义 css 建议在博客目录 `assets\css\` 下建立 `_custom.scss`

 1. 由于 #62 有使用者说希望提供宽屏选项，考虑到设备的屏幕分辨率越来越大，这个 issue 确实有处理的必要，所以调整后提供了两个选项
     ```
     # Page style ("normal", "wide", ...)
     pageStyle = "normal"
     ```
    `normal` 比 LoveIt 原宽度更大了一点，`wide` 则是更宽。
    要修改成原来的窄布局的话，可以参照 `wide` 模式的实现，在 `_custom.scss` 里面自定义，然后修改 config 里的 pageStyle 值
 2. 自定义字体也属于 css 的范畴，你可以参考以下我的博客
     - 定义字体 [Lruihao/hugo-blog/assets/css/_custom.scss#L173-L184](https://github.com/Lruihao/hugo-blog/blob/be0c14389544ca0ab9be697e01af78c25ac139a3/assets/css/_custom.scss#L173-L184)
     - 使用字体 [Lruihao/hugo-blog/assets/css/_custom.scss#L75](https://github.com/Lruihao/hugo-blog/blob/be0c14389544ca0ab9be697e01af78c25ac139a3/assets/css/_custom.scss#L75)
    如果要全局使用字体的话就在 `_variables.scss` 修改或者在 `_override.scss` 覆盖 scss 变量 `$global-font-family` 的值

**Reply by Xm798** [reply]
太感谢您耐心的解答了！我明天按照如上说明尝试一下。
另外还遇到一个新的问题，就是主页上展示的文章作者字号，是哪一段样式控制的呢？找到了标题和副标题，貌似没有找到类似于 author 或者 meta 之类的样式。

**Reply by Lruihao** [reply]
> 太感谢您耐心的解答了！我明天按照如上说明尝试一下。 另外还遇到一个新的问题，就是主页上展示的文章作者字号，是哪一段样式控制的呢？找到了标题和副标题，貌似没有找到类似于 author 或者 meta 之类的样式。
```css
.author {
    // 
}
.post-meta {
    // 
}
```
@Xm798  你可以打开 F12 找每个元素的 class 然后在 `_custom.scss` 修改即可

**Reply by Xm798** [reply]
学到了！忘记还有 F12 了😂 感谢！！

**Comment by Lruihao** [comment]
@Xm798 我添加了一个 pageStyle 选项 `narrow` (`narrow` 保留和 LoveIt 一致的宽度)
你更新到 RC 版就行了 https://github.com/Lruihao/FixIt/commit/000e0d235934b098cf24c2b5d9d809d106a21933

[pageStyle文档](https://fixit.lruihao.cn/theme-documentation-basics/#page-style)

**Reply by Xm798** [reply]
十分感谢！昨天找了一会没找到在哪里修改，现在又学到了
